experimental/origin-dapp2/package.json

```diff
@@ -35,6 +35,7 @@
     "apollo-link": "^1.2.8",
     "babel-plugin-module-resolver": "^3.2.0",
     "blueimp-load-image": "^2.20.1",
+    "big-number": "^2.0.0",
```

Comment:
Sorry to be a pain... looks like web3 is in fact used throughout dapp2 haha. Perhaps we can remove this dependency and go back to the old way of using web3.utils, for consistency?

Reply:
Sure no problem :)